ContestUtil/src/org/icpc/tools/contest/util/ImagesGenerator.java

```diff
@@ -434,8 +437,11 @@ private static File getDefaultFile(File folder, String property) {
 	}
 
 	public void generateOrganizationLogos() {
-		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(ICON, true),
-				new ImageSpec(TILE, true) };
+		ImageSpec[] spec = new ImageSpec[] { new ImageSpec(DESKTOP, false), new ImageSpec(TILE, true) };
+		spec = Arrays.copyOf(spec, spec.length + ICONS.length);
```

Comment:
nit: if you record the original length in a variable before the array copy, the indexing below will be nicer

Reply:
Smart, done!